clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -20,6 +20,7 @@ export const COPYBOOKS_FOLDER: string = ".copybooks";
 export const GITIGNORE_FILE: string = ".gitignore";
 export const LANGUAGE_ID = "COBOL";
 export const PROCESS_DOWNLOAD_ERROR_MSG = "Some copybooks could not be located. Ensure your configuration contains correct paths to copybooks, including nested copybooks. Missing copybooks: ";
+export const DOWNLOAD_QUEUE_LOCKED_ERROR_MSG = "Your queue was locked due to invalid credentials.";
```

Comment:
Maybe, it would be more descriptive to say something like "Copybook downloading stopped due to invalid credentials". I would recommend to discuss it with @zimlu02 or @zeibura 
Besides that, as I remember, there should be no dots at the end of the info messages

Reply:
I've tried to make it more user friendly:
![image](https://user-images.githubusercontent.com/60617592/94131164-de1bd100-fe5d-11ea-9416-c5c18172d2bc.png)
@zimlu02, @zeibura what do you think?